src/include/nanobench.h

```diff
@@ -2991,6 +3014,16 @@ double Result::maximum(Measure m) const noexcept {
     return *std::max_element(data.begin(), data.end());
 }
 
+std::string const& Result::context(std::string const& variableName) const noexcept {
+    auto context = mConfig.mContext;
```

Comment:
This creates a copy of the map, please use `auto const& context = mConfig.mContext` instead

Reply:
Stupid mistake :see_no_evil: